shopper-app/src/components/AccountCardBox.jsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { PageContext } from '@/context/Page';
+import { useRouter } from 'next/router';
+
+export default function OutlinedCard({ title, list }) {
```

Comment:
name of file and default export should be the same name. Also, `src/components/AccountCards.jsx` also has  a default export with the same name. Someone who has not worked on this code should know what these components are based off of their name